Source/Evolve/WarpXComputeDt.cpp

```diff
@@ -32,15 +33,31 @@
 void
 WarpX::ComputeDt ()
 {
+    // Check if any species is using supercycling
+    bool do_supercycling = false;
+    int nSpecies = mypc->nSpecies();
```

Comment:
```suggestion
    const int nSpecies = mypc->nSpecies();
```
Hi @archermarx ! I think that the absence of `const` here is causing the `clang-tidy` CI test to fail.

Reply:
Thanks! Fixed.